src/components/dropdown/DropdownItem.vue

```diff
@@ -4,13 +4,16 @@
         v-else-if="!custom && !hasLink"
         class="dropdown-item"
         :class="anchorClasses"
-        @click="selectItem">
+        @click="selectItem"
+        role="menuitem"
+        :tabindex="tabIndex">
         <slot/>
     </a>
     <div
         v-else
         :class="itemClasses"
-        @click="selectItem">
+        @click="selectItem"
+        :tabindex="tabIndex">
```

Comment:
This DIV is focusable, but it's missing an interactive role to be announced in screen readers. I'd recommend converting it to a button element (ideal), as it's the easiest thing to do. You could add`role="button"` instead of converting it to a real button, but you'd also need a keyboard event since DIVs don't fire click events like native buttons do.